account_financial_report_qweb/report/aged_partner_balance_xlsx.py

```diff
@@ -15,8 +15,10 @@ def __init__(self, name, table, rml=False, parser=False, header=True,
         super(AgedPartnerBalanceXslx, self).__init__(
             name, table, rml, parser, header, store)
 
-    def _get_report_name(self):
-        return _('Aged Partner Balance')
+    def _get_report_name(self, objects):
+        report = objects
```

Comment:
The renaming of the parameter would avoid this line

Reply:
@Cedric-Pigeon I have fixed according to your comments.